randovania/games/am2r/logic_database/Distribution Center.txt

```diff
@@ -571,12 +572,13 @@ Extra - minimap_data: [{'x': 66, 'y': 42}, {'x': 66, 'y': 43}, {'x': 66, 'y': 44
           # IBJ
           Infinite Bomb Jumping (Advanced) and Can Use Bombs
 
-> Pipe to Distribution Facility Tower East (Middle); Heals? False
+> Pipe to Distribution Facility Tower East (Top); Heals? False
   * Layers: default
-  * Teleporter to Distribution Facility Tower East/Pipe to Distribution Facility Tower West (Top)
+  * Teleporter to Distribution Facility Tower East/Pipe to Distribution Facility Tower West (Middle); Custom name: Distribution Center - Distribution Facility Tower West Top Pipe
```

Comment:
This pipe and some others don't seem to have made it into the test file .jsons, is there a reason for this?

Reply:
Unsure what you mean here?
The full transport shuffle definitely includes them:
![grafik](https://github.com/randovania/randovania/assets/38186597/141400c2-f990-44e7-9eed-f8b47675349f)
![grafik](https://github.com/randovania/randovania/assets/38186597/c9dc8bc7-6356-4f68-96e5-d47a2483419d)

The custom name is only ever shown on the RDV UI.

Reply:
In test files/patcher_data/am2r/am2r the worlds.json rename the gravity area pipes for each, but you also renamed this pipe to (top) so I'm asking if that also needs to be updated in those test files.

Reply:
Not the gravity pipes that were renamed, but the *areas* of the gravity pipes.
Here I only changed the names of the nodes. And am2r does not show you the name of nodes in-game.